TweetMate/src/main/java/com/ybsystem/tweetmate/utils/ExceptionUtils.java

```diff
@@ -25,19 +27,38 @@ private ExceptionUtils() {
     public static String getErrorMessage(Throwable t) {
 
         if (!(t instanceof TwitterException)) {
-            return "読み込みエラー";
+            if (PrefSystem.getLanguage().equals("en")) {
+                return "Failed to load…";
+            } else {
+                return "読み込みエラー";
+            }
         }
 
         TwitterException e = (TwitterException) t;
 
         if (e.isCausedByNetworkIssue()) {
-            return "ネットワーク接続エラー";
+            if (PrefSystem.getLanguage().equals("en")) {
+                return "Network error…";
+            } else {
+                return "ネットワーク接続エラー";
+            }
         }
 
         if (e.exceededRateLimitation() || e.getErrorCode() == 88) {
-            return "API制限に到達しました。\n"
-                    + e.getRateLimitStatus().getSecondsUntilReset() / 60 + "分"
-                    + e.getRateLimitStatus().getSecondsUntilReset() % 60 + "秒後に解除されます。";
+            if (PrefSystem.getLanguage().equals("en")) {
+                return "API limit reached.\n"
+                        + "It will be lifted after "
+                        + e.getRateLimitStatus().getSecondsUntilReset() / 60 + "m"
+                        + e.getRateLimitStatus().getSecondsUntilReset() % 60 + "s.";
+            } else {
+                return "API制限に到達しました。\n"
+                        + e.getRateLimitStatus().getSecondsUntilReset() / 60 + "分"
+                        + e.getRateLimitStatus().getSecondsUntilReset() % 60 + "秒後に解除されます。";
+            }
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "対象が存在しません。" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1192)